### PR TITLE
build: add license and readme to built library

### PR DESCRIPTION
**Pull request by baptistechardon**
The license, README, and CHANGELOG were missing from the built library. Added it to build step.

## PR Checklist

Please check if your PR fulfills the following requirements:

- [X] The commit message follows [our guidelines](https://github.com/btroncone/ngrx-store-localstorage/blob/master/CONTRIBUTING.md#commit):
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [X] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Other... Please describe:

## Current Behavior

Issue Number: [215]([https://github.com/btroncone/ngrx-store-localstorage/issues/215])

## Breaking Change?

- [ ] Yes
- [X] No

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -59,6 +63,7 @@
         "@types/node": "^14.17.0",
         "@typescript-eslint/eslint-plugin": "5.3.0",
         "@typescript-eslint/parser": "5.3.0",
+        "copyfiles": "^2.4.1",
```
Comment (baptistechardon) [inline package.json:66]: New dev dependency to copy the files

File: projects/lib/package.json [hunk 2]
```diff
@@ -9,5 +9,6 @@
     "dependencies": {
         "deepmerge": "^4.2.2",
         "tslib": "^2.3.0"
-    }
+    },
+    "license": "MIT"
```
Comment (baptistechardon) [inline projects/lib/package.json:13]: The MIT license was specified in the main package.json, and was the license displayed on the npm registry in previous versions.

File: package.json [hunk 3]
```diff
@@ -1,33 +1,17 @@
...
-        "build_dist": "npm run clean && ng build lib --configuration production",
+        "build_dist": "npm run clean && ng build lib --configuration production && npm run copy_lib_assets",
         "clean": "rimraf dist",
         "lint": "ng lint --fix",
         "ng": "ng",
         "pretest": "npm run lint",
         "test": "nyc ts-node -O '{\"module\": \"commonjs\"}' node_modules/jasmine/bin/jasmine",
-        "prepare": "husky install"
+        "prepare": "husky install",
+        "copy_lib_assets": "copyfiles README.md CHANGELOG.md LICENSE ./dist/lib/"
```
Comment (BBlackwo) [inline package.json:13]: I simplified this into a single script

File: projects/lib/package.json [hunk 4]
```diff
@@ -1,6 +1,24 @@
 {
     "name": "ngrx-store-localstorage",
     "version": "13.0.0",
+    "description": "State and local storage syncing for @ngrx/store",
```
Comment (BBlackwo) [inline projects/lib/package.json:4]: Added a bunch of the other missing fields to the lib package.json

**Review by BBlackwo — APPROVED** [review]
@baptistechardon thanks so much for raising this issue and PR. I've added some additional fields to the lib package.json and going to merge it now.

**Pull request merged**